o1vm/src/mips/column.rs

```diff
@@ -14,14 +14,22 @@ use strum::EnumCount;
 
 use super::{ITypeInstruction, JTypeInstruction, RTypeInstruction};
 
-pub(crate) const MIPS_HASH_COUNTER_OFFSET: usize = 80;
-pub(crate) const MIPS_IS_SYSCALL_OFFSET: usize = 81;
-pub(crate) const MIPS_READING_PREIMAGE_OFFSET: usize = 82;
-pub(crate) const MIPS_BYTES_READ_OFFSET: usize = 83;
-pub(crate) const MIPS_PREIMAGE_LEFT_OFFSET: usize = 84;
-pub(crate) const MIPS_PREIMAGE_BYTES_OFFSET: usize = 85;
-pub(crate) const MIPS_HAS_N_BYTES_OFFSET: usize = 89;
-pub(crate) const MIPS_CHUNK_BYTES_LENGTH: usize = 4;
+/// The number of hashes performed so far in the block
+pub(crate) const MIPS_HASH_COUNTER_OFF: usize = 80;
+/// The number of bytes of the preimage that have been read so far in this hash
+pub(crate) const MIPS_BYTE_COUNTER_OFF: usize = 81;
+/// A flag indicating whether the preimage has been read fully or not
+pub(crate) const MIPS_END_OF_PREIMAGE_OFF: usize = 82;
+/// The number of preimage bytes processed in this step
+pub(crate) const MIPS_NUM_BYTES_READ_OFF: usize = 83;
+/// The at most 4-byte chunk of the preimage that has been read in this step
+pub(crate) const MIPS_PREIMAGE_CHUNK_OFF: usize = 84;
```

Comment:
Nit: (might be only for me) I think it might be worth to add some small examples on what the registers at these offsets are supposed to embed? Like integers, bytes, etc. Just to know, when we read this, where we are supposed to find them in the code.